src/components/plugins/MapQuestionnaire.js

```diff
@@ -0,0 +1,199 @@
+import Button from 'react-bootstrap/lib/Button';
+import BaseCommentForm from '../BaseCommentForm';
+import CommentDisclaimer from "../CommentDisclaimer";
+import FormControl from 'react-bootstrap/lib/FormControl';
+import FormGroup from 'react-bootstrap/lib/FormGroup';
+import React from 'react';
+import {injectIntl, FormattedMessage} from 'react-intl';
+import {alert} from '../../utils/notify';
+
+
+class MapQuestionnaire extends BaseCommentForm {
```

Comment:
Yes they do, because they are tied to specific source. They won't be reused ever, as the source is not updated.